ui/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/SystemComment.scala

```diff
@@ -0,0 +1,7 @@
+package pl.touk.nussknacker.ui.process.repository
+
+import pl.touk.nussknacker.ui.listener.Comment
+
+case class SystemComment(value: String) extends Comment
+
+case class UpdateProcessComment(value: String) extends Comment
```

Comment:
`UpdateProcessComment` is a `SystemComment`. Can u add inheritance that we can see all comments created by system?

Reply:
changed it, but would like to discuss this inheritance further because I'm not certain how it would work best

Reply:
you have `UserComment` in `SystemComment.scala`

Reply:
Also this override of comment/value on each three levels of inheritance is very complicated. Can u simplify that?

Reply:
ad 1 moved to seperate file, thanks 
ad2 didn't notice redundancies while changing the logic, should be better now please let me know if it's ok